rmc/utils/mpc.py

```diff
@@ -517,44 +504,191 @@ def _get_min_aic_glm(
     )
     logger.info("Coefficients: %s", overall_min_model.params)
 
-    logger.info("Saving model as pickle...")
-    model_path = mpc_model_pkl_path(freeze)
-    with hl.hadoop_open(model_path, "wb") as p:
-        pickle.dump(overall_min_model, p, protocol=pickle.HIGHEST_PROTOCOL)
+    return overall_min_model
 
-    logger.info(
-        "Annotating gnomAD variants with fitted score and writing to gnomad_fitted_score path..."
-    )
-    ht = ht.filter(ht.pop_v_path == 1)
-    ht = calculate_fitted_scores(ht=ht, freeze=freeze)
-    ht.write(gnomad_fitted_score_path(freeze=freeze), overwrite=overwrite)
+
+def run_regressions(
+    use_model_formula: bool = False,
```

Comment:
is this just to make it easier for our team tor run just one formula (if we choose one specific formula to go with for v2)?

Reply:
Yes